packages/cli/src/lib/transform.ts

```
@@ -78,26 +105,67 @@
+    const errors = parseErrors(codemod, stdout);
+    if (options.logStatus && errors.length > 0) {
+      errors.forEach(({ transform, filename, summary }) => {
+        error(
+          `Error applying codemod [codemod=${transform}, path=${filename}, summary=${summary}]`,
+        );
+      });
+    }
+    return errors;
+  } else {
+    // Use the original synchronous approach if no progress callback
+    const stdout = execFileSync(command[0]!, command.slice(1), {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)